modules/hardware-qualcomm.nix

```diff
@@ -70,6 +75,13 @@ in
         quirks.fb-refresher.enable = true;
       };
     }
+    {
+      # not sure if right:
+      mobile = mkIf cfg.qualcomm-sdm845.enable {
+        system.system = "aarch64-linux";
+        quirks.fb-refresher.enable = true;
```

Comment:
The fb-refresher is definitely not right as long as DRM only.

I guess it will all depend on whether I'm successful in making an fbdev variant.

Reply:
ok, dropping for now